frontend/src/components/dashboard/dashboard.tsx

```diff
@@ -29,7 +38,7 @@ const Dashboard: FC = () => {
     <div className={styles.dashboard}>
       <div className={styles.headerWrapper}>
         <div className={styles.header}>
-          <h1 className={styles.headingText}>Courses</h1>
+          <h1 className={styles.headingText}>Course</h1>
```

Comment:
```suggestion
          <h1 className={styles.headingText}>Courses</h1>
```
🤔

Reply:
It's "Course" in Figma, that's the reason it was changed :sweat_smile:

Reply:
should be Courses